{{cookiecutter.repo_name}}/package.json

```diff
@@ -444,7 +444,6 @@
       "at-rule-name-case": "lower",
       "at-rule-name-space-after": "always",
       "at-rule-no-vendor-prefix": true,
-      "stylelint-disable-reason": "always-before",
```

Comment:
Thinking about it, I see their logic completely (they they can only enforce that _there is a comment above it_, not an actual _reason_). I plan to always add a reason either way. :)